dv/uvm/ibex_dv.f

```diff
@@ -27,7 +27,7 @@
 ${PRJ_DIR}/ibex/rtl/ibex_multdiv_fast.sv
 ${PRJ_DIR}/ibex/rtl/ibex_prefetch_buffer.sv
 ${PRJ_DIR}/ibex/rtl/ibex_fetch_fifo.sv
-${PRJ_DIR}/ibex/rtl/ibex_register_file_ff.sv
+${PRJ_DIR}/ibex/rtl/ibex_register_file_fpga.sv
```

Comment:
I think we do not want to use the new FPGA Regfile by default. Thus I suggest to undo this change.

Reply:
Done. Thank you!